css/style.css

```diff
@@ -0,0 +1,191 @@
+  margin-top:13px !important;
+  margin-bottom:4px !important;
+  text-transform:uppercase !important;
+  color: #515151 !important;
+  font-family:sans-serif !important;
+  font-size:13px !important;
+  text-align:left;
+  display:inline-block;
+  display:-webkit-inline-box;
+  height:18px !important;
+}
+
+.HPS_gform_card_icon_container {
+  height:42px !important;
+}
+
+.gform_wrapper #HPS_secure_ach input {
+    padding:15px !important;
+    color:#8c8c8c !important;
+}
+
+.gform_wrapper #HPS_secure_ach input::-webkit-input-placeholder,
+.gform_wrapper #HPS_secure_ach input::-moz-placeholder,
+.gform_wrapper #HPS_secure_ach input:-ms-input-placeholder,
+.gform_wrapper #HPS_secure_ach input:-moz-placeholder {
+  color:#8c8c8c !important;
+  font-family:sans-serif !important;
+}
+
+.gform_wrapper #HPS_secure_ach input,
+.gform_wrapper #HPS_secure_ach select {
+  font-size: small;
+  height:48px !important;
+  border:1px solid #b5b5b5 !important;
+  text-transform:uppercase !important;
+  background-color:transparent;
+  border-radius:0px;
+  color:#8c8c8c !important;
+  font-family:sans-serif !important;
+  letter-spacing:1.5 !important;
+}
+
+.gform_wrapper #HPS_secure_ach select {
+  border:0 !important;
+  outline:1px solid #b5b5b5;
+  padding:10px;
+  font-size:small !important;
+}
+
+.gform_wrapper #HPS_secure_ach input:hover,
+.gform_wrapper #HPS_secure_ach input:focus {
+  border:1px solid #3989e3 !important;
+  outline:0 !important;
+  -webkit-transition: border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;
+  -o-transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
+  transition: border-color ease-in-out .15s,box-shadow ease-in-out .15;
+  color:#515151 !important;
+  letter-spacing:1.5 !important;
+}
+
+.gform_wrapper #HPS_secure_ach select:hover,
+.gform_wrapper #HPS_secure_ach select:focus {
+  border:0 !important;
+  outline:1px solid #3989e3 !important;
+  -webkit-transition: border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;
+  -o-transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
+  transition: border-color ease-in-out .15s,box-shadow ease-in-out .15;
+  color:#515151 !important;
+}
+
+.gform_wrapper #HPS_secure_ach input[type=submit] {
+  background-color:#3989e3;
+  color:white;
+  padding:10px;
+  font-size:14px;
+  font-weight:400;
+  vertical-align:middle;
+  -ms-touch-action: manipulation;
+  touch-action: manipulation;
+  cursor: pointer;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select:none;
+  user-select:none;
+  border-radius:0px;
+  border:1px solid #2e6da4;
+}
+
+/*error message*/
+.gform_wrapper .validation_message,
+.gform_wrapper div.validation_error,
+.gfield_error
+ {
+  font-family:sans-serif !important;
+  font-weight:400 !important;
+  font-size:17px !important;
+  padding:10px !important;
+  background-color:transparent !important;
+}
+
+
+div.gfield_description.validation_message {
```

Comment:
I asked myself the same thing, but I don't know if  it is possible. If there is a way to apend a class or ID to our validation/error messages inside of a function let me know.